skills/csharp/experimental/itsmskill/Controllers/ServiceNow/ServiceNowControllerBase.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace ITSMSkill.Controllers.ServiceNow
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using ITSMSkill.Models.ServiceNow;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Bot.Builder;
+
+    public abstract class ServiceNowControllerBase : Controller
```

Comment:
I don't find it is necessary to use ServiceNowControllerBase and ServiceNowController. We could just DeserializeObject and call Receive in Controller. One could even wrap it to a template function.